frontend/src/routes/Supper/AddFoodItem/index.tsx

```diff
@@ -1,3 +1,4 @@
+/*eslint @typescript-eslint/no-explicit-any : off */
```

Comment:
should we be disabling no-explicit-any for the whole page like that, or by a case-by-case basis. feels like defeating the purpose

Reply:
case by case is a little hard coz certain chunks of code disallow eslint comments